sunpy/net/tests/test_attr.py

```diff
@@ -344,11 +345,20 @@ def test_attr_numbes():
 
 
 def test_attr_iterable_length():
+
     # not iterable
-    with pytest.raises(ValueError):
+
+    error_message1 = "The value is not iterable or just a string"
+    pattern_1_1 = re.compile(error_message1, re.DOTALL)
+
+    with pytest.raises(ValueError, match=pattern_1_1):
         attr.Attr.update_values({GenericClient: {Instrument: 'AIA'}})
+
     # too many items
-    with pytest.raises(ValueError):
+
+    error_message2 = "Error now"
```

Comment:
There is no need to define this variable, you can just pass the string directly to match.